tests/Drupal/Commenting/DocCommentUnitTest.1.inc

```diff
@@ -7,3 +7,36 @@
  * A comprehensive guide with examples is available at:
  * @link http://example.com Test link @endlink.
  */
+
+/**
+ * Short comment with acceptable ending punctuation that does not need fixing!
+ */
+
+/**
+ * It is an unlikely comment, but question marks are acceptable, and why not?
+ */
+
+/**
+ * Brackets are also an allowed ending with no warning()
+ */
+
+/**
+ * This is a short comment with no full stop that can be fixed automatically
+ */
+
+/**
+ * A comment ending in digits is fixable, but the number of these in Core is 0
```

Comment:
lol, you are creative ... these tests make me smile :)

Reply:
It's important not to be dull, and help the maintainer enjoy the work.